src/components/offer-card-square/OfferCardSquare.tsx

```diff
@@ -0,0 +1,81 @@
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+import LanguageIcon from '@mui/icons-material/Language'
+import IconButton from '@mui/material/IconButton'
+import TurnedInNot from '@mui/icons-material/TurnedInNot'
+
+import { t } from 'i18next'
+import AppCard from '~/components/app-card/AppCard'
+import ImgTitleDescription from '~/components/img-title-description/ImgTitleDescription'
+import AppChip from '../app-chip/AppChip'
+import AppRatingMobile from '../app-rating-mobile/AppRatingMobile'
```

Comment:
Change path because we use an alias and all import need to be in one style.

Reply:
fixed